charts/controls/Controls.tsx

```diff
@@ -976,7 +956,7 @@ export class ControlsFooterView extends React.Component<{
 
         const timelineElement = hasTimeline && (
             <div className="footerRowSingle">
-                <TimelineControl chart={this.props.controls.props.chart} />
+                <TimelineControl chart={chart} activeTab={chart.tab} />
```

Comment:
Nit: you can just pass chart and then access `.tab` inside TimelineControl.

Reply:
I did this to cause a tab change to force a rerender and `componentDidUpdate`, as used below.

Reply:
Ah interesting!